src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -36,6 +39,8 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "md", "lg",
 
   &.pf-m-horizontal {
     --pf-c-description-list__group--GridTemplateColumns: var(--pf-c-description-list--m-horizontal__group--GridTemplateColumns);
+
+    @include pf-build-css-variable-stack("--pf-c-description-list--m-horizontal__term--width", "--pf-c-description-list__term--width", $pf-c-description-list--breakpoint-map);
```

Comment:
horizontal variation broke. looks like you need to define `--pf-c-description-list--m-horizontal__term--width`?